src/applications/personalization/profile/tests/fixtures/users/user-36.json

```diff
@@ -39,7 +39,18 @@
       "inProgressForms": [],
       "prefillsAvailable": [],
       "vet360ContactInformation": {
-        "email": null,
+        "email": {
```

Comment:
We are using this mock user in Notification Settings tests so they should have an email address by default. Making this change does not affect any existing tests that use this user.